include/nbl/video/IPhysicalDevice.h

```diff
@@ -69,15 +69,17 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
         //
         struct APIVersion
         {
-            uint32_t major : 5;
-            uint32_t minor : 5;
-            uint32_t patch : 22;
+            uint32_t major : 8;
+            uint32_t minor : 8;
+            uint32_t subminor : 8;
+            uint32_t patch : 8;
 
             inline auto operator <=> (uint32_t vkApiVersion) const { return vkApiVersion - VK_MAKE_API_VERSION(0, major, minor, patch); }
             inline auto operator <=> (const APIVersion& other) const 
             {
                 if(major != other.major) return static_cast<uint32_t>(other.major - major);
                 if(minor != other.minor) return static_cast<uint32_t>(other.minor - minor);
+                if(subminor != other.subminor) return static_cast<uint32_t>(other.subminor - subminor);
```

Comment:
this is an important change I've made, didn't want `vkConformanceVersion` floating around in our API